projects/ccf-eui/src/app/app-web-component.component.ts

```diff
@@ -38,6 +38,10 @@ export class AppWebComponent extends BaseWebComponent {
 
   @Input() useRemoteApi: string | boolean;
   @Input() remoteApiEndpoint: string;
+  @Input() theme: string;
+  @Input() header: string | boolean;
+  @Input() homeUrl: string;
+  @Input() logoTooltip: string;
```

Comment:
loginDisabled needs to be a web component input... How did you expect folks to disable login without it????